habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -240,6 +247,20 @@ def _init_train(self):
 
         self._init_envs()
 
+        if self.using_velocity_ctrl:
+            self.policy_action_space = ActionSpace(
```

Comment:
Is the task action not able to return this properly?  If not we need to fix that interface instead of patching it in the trainer

Reply:
If POSSIBLE_ACTIONS is just ['VELOCITY_CONTROL'], then self.envs.action_space has only one action, but the policy technically has two, linear and angular velocity, so without this fix, the policy would only have one action. I don't know how to fix this other than doing something like ['LINEAR_VELOCITY','ANGULAR_VELOCITY'] and then coding each of the two separately, but I don't know how it would be possible to implement these move_fns independently from one another, as you can't just translate and then rotate (or vice versa).

Reply:
Hmm, I guess the VELOCITY_CONTROL really needs to be able to return a dictionary of spaces?  @mathfac Is that supported? I remember you working on something like this for the EQA answer action

Reply:
@mathfac bumping this, do you know if returning a spaces dict from one action is supported?

Reply:
I checked, it is supported.  You can see its use in the Teleport action just above this.  Habitat baselines will need a bit of working on to get things working on its side however.

Reply:
That working on habitat-baselines is likely outside the scope of this PR (just took a look and doing this in a general will take a fair amount of thought).  For now, let's just return the correct action space in the action and then this can be
```
self.policy_action_space = self.envs.action_spaces[0]['VELOCITY_CONTROL']
```

Reply:
@erikwijmans Could you clarify what you mean by "return the correct action space in the action"?